src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -75,7 +75,11 @@ func DeployCmd(noMetrics *bool) *cobra.Command {
 			if err := assets.WriteLocalAssets(manifest, opts, hostPath); err != nil {
 				return err
 			}
-			return maybeKcCreate(dryRun, manifest)
+			ret := maybeKcCreate(dryRun, manifest)
```

Comment:
I'm running `maybeKcCreate()` first (and capturing it's output) because it (or, actually, the k8s commands it runs) outputs several lines while doing the deploy, and I want the UI info output to follow that.

Can definitely move the output code into `maybeKcCreate()` -- I thought you'd said to duplicate it in each deploy function because you wanted to keep `maybeKcCreate()` only doing the generation and not pass in any superfluous context

FWIW, I think ideally the output would happen from the calling code -- where it is now -- but each of these DeployCmd functions would be refactored to be a single parameterized function. That seems like a fair bit of tweaking, though, and it seems Cobra wants these to be individual, so not going to bother for now -- I'll just polute `maybeKcCreate()` a bit and avoid adding any more code duplication